concourse/pipelines/destroy.yml

```diff
@@ -1,7 +1,7 @@
 ---
 resources:
```

Comment:
This is missing the `resource_types` block needed by all the s3 and semver resources. (Fix in #51)